neuralprophet/forecaster.py

```diff
@@ -1370,8 +1393,12 @@ def plot_last_forecast(
         """
         if self.max_lags == 0:
             raise ValueError("Use the standard plot function for models without lags.")
-        if isinstance(fcst, dict):
-            log.error("Received more than one DataFrame. Use a for loop for many dataframes.")
+        fcst_dict, received_unnamed_df, received_dict = df_utils.convert_df_to_dict_or_copy_dict(fcst)
+        if len(fcst_dict) > 1:
+            raise Exception(
+                "Many time series are present in the pd.DataFrame (more than one ID). Use a for loop with groupby for plotting many time series."
```

Comment:
We could allow the user to specify the ID they want to plot, so they don't have to do the groupby themselves